app/org/sagebionetworks/bridge/models/accounts/Phone.java

```diff
@@ -0,0 +1,72 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.Transient;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+import com.google.i18n.phonenumbers.PhoneNumberUtil.PhoneNumberFormat;
+import com.google.i18n.phonenumbers.Phonenumber.PhoneNumber;
+
+@Embeddable
+public class Phone {
+    
+    public static final boolean isValid(Phone phone) {
+        try {
+            PhoneNumber phoneNumber = PHONE_UTIL.parse(phone.getNumber(), phone.getRegionCode());
+            return PHONE_UTIL.isValidNumber(phoneNumber);
+        } catch (Exception e) {
+        }
+        return false;
+    }
+    
+    private static final PhoneNumberUtil PHONE_UTIL = PhoneNumberUtil.getInstance();
+    private String number;
+    private String regionCode;
+    
+    // For Hibernate. 
+    Phone() {
+    }
+    
+    @JsonCreator
+    public Phone(@JsonProperty("number") String number, @JsonProperty("regionCode") String regionCode) {
+        this.number = number;
+        this.regionCode = regionCode;
+    }
```

Comment:
Why couldn't Jackson use the default constructor and use getters and setters for attributes?